Shopping List/Base.lproj/Main.storyboard

```diff
@@ -1,24 +1,172 @@
+                                            <nil key="textColor"/>
+                                            <nil key="highlightedColor"/>
+                                        </label>
+                                    </subviews>
+                                </view>
+                                <constraints>
+                                    <constraint firstItem="8aQ-wU-mKd" firstAttribute="top" secondItem="N9A-W4-l4X" secondAttribute="bottom" constant="3" id="1Lh-K4-Jbm"/>
+                                    <constraint firstItem="8aQ-wU-mKd" firstAttribute="leading" secondItem="cY6-2N-8fl" secondAttribute="leadingMargin" constant="18" id="55l-Hg-3dx"/>
+                                    <constraint firstAttribute="trailingMargin" secondItem="8aQ-wU-mKd" secondAttribute="trailing" constant="17" id="9dR-Y4-5M8"/>
+                                    <constraint firstItem="II9-g7-Ig0" firstAttribute="top" secondItem="8aQ-wU-mKd" secondAttribute="bottom" constant="3" id="CBA-4p-6rs"/>
+                                    <constraint firstItem="II9-g7-Ig0" firstAttribute="leading" secondItem="cY6-2N-8fl" secondAttribute="leading" id="LRN-5H-oPo"/>
+                                    <constraint firstAttribute="trailing" secondItem="II9-g7-Ig0" secondAttribute="trailing" id="Rju-zf-uXF"/>
+                                    <constraint firstItem="N9A-W4-l4X" firstAttribute="top" secondItem="cY6-2N-8fl" secondAttribute="top" id="Sqj-3J-teZ"/>
+                                    <constraint firstAttribute="trailing" secondItem="N9A-W4-l4X" secondAttribute="trailing" id="WJl-Kk-Q57"/>
+                                    <constraint firstItem="II9-g7-Ig0" firstAttribute="width" secondItem="cY6-2N-8fl" secondAttribute="width" id="Yd9-Dc-f88"/>
+                                    <constraint firstItem="II9-g7-Ig0" firstAttribute="top" secondItem="8aQ-wU-mKd" secondAttribute="bottom" constant="3" id="bFF-o9-3to"/>
+                                    <constraint firstAttribute="bottom" secondItem="II9-g7-Ig0" secondAttribute="bottom" id="dJ2-cF-UXB"/>
+                                    <constraint firstAttribute="trailing" secondItem="N9A-W4-l4X" secondAttribute="trailing" id="h0q-UE-Cet"/>
+                                    <constraint firstItem="N9A-W4-l4X" firstAttribute="leading" secondItem="cY6-2N-8fl" secondAttribute="leading" id="hKB-9z-zh0"/>
+                                </constraints>
+                                <connections>
+                                    <outlet property="addedLabel" destination="N9A-W4-l4X" id="b1f-eD-ze5"/>
+                                    <outlet property="imageView" destination="8aQ-wU-mKd" id="dpX-zj-JwI"/>
+                                    <outlet property="nameLabel" destination="II9-g7-Ig0" id="hgy-pZ-Hha"/>
+                                    <segue destination="qVo-eJ-y7n" kind="show" identifier="cellSegue" id="fAA-eX-w8p"/>
```

Comment:
You don't need a segue from `collectionViewCell` to `detailView`. Just delete it